bolt-design-system/bolt
# Stylelint CSS props order

## Jira

http://vjira2:8080/browse/BDS-356

## Summary

Turns on stylelint's CSS Properties Order based on Bolt's convention.

## Details

Added and re-organized CSS props in the stylelint config, and updated all SCSS files based on that.

## How to test

1. Run the branch locally.
2. Click on every single page for any visual regression.
3. Inspect the CSS changes and make sure they are only re-ordering the props, make sure nothing was removed by mistake.
